packages/docusaurus/src/client/preload.ts

```diff
@@ -7,6 +7,7 @@
 
 import routes from '@generated/routes';
 import {matchRoutes} from 'react-router-config';
+import { uniq } from 'lodash';
```

Comment:
This is client code. Please hand-roll your implementation (should be fairly trivial)

Reply:
checked the `component.preload` implementation, `uniq` is not necessary. preload won't actually run twice.